rockspec/apisix-master-0.rockspec

```diff
@@ -58,6 +58,7 @@ dependencies = {
     "resty-redis-cluster = 1.02-4",
     "lua-resty-expr = 1.0.0",
     "graphql = 0.0.2",
+    "lua-resty-dns-client = 5.1.1-2",
```

Comment:
I think it would be safer if we keep two dns client libraries, and use lua-resty-dns-client by default but provide an option to switch to the old one.